test/e2e/pensions-helpers.js

```diff
@@ -170,6 +170,116 @@ function completeDependentAddressInfo(client, data, index) {
   }
 }
 
+function completeNetWorthInfo(client, data) {
```

Comment:
Some of these functions are also in [your previous PR](https://github.com/department-of-veterans-affairs/vets-website/pull/6079/files), by the by. I'm assuming you branched off that other branch, so you can either do some git magic to remove first two commits (but don't ask me how!), or deal with the merge conflict later.

Don't make no nevermind to me either way! Just thought I'd do my civic duty and point it out.